### ISSUE TITLE
`request.text()` fails to produce parseable contents in 1.10-pre5

### ISSUE THREAD

**Issue by ZipBrandon** [remix-run/remix#4967]
### What version of Remix are you using?

1.10-pre5

### Steps to Reproduce

This results in  https://remix.run/docs/en/v1/pages/faq#how-can-i-have-structured-data-in-a-form being an invalid approach.

```ts
export const action: ActionFunction = async ({
  request,
  context,
  params,
}): Promise<ActionData | null> => { 
let formQueryString = await request.text();
  console.log({ formQueryString }); 
}
```
  will produce something to this effect: 
```js
{
  formQueryString: '------WebKitFormBoundaryccYLQ3im9udgCOSx\r\n' +
    'Content-Disposition: form-data; name="__rvfInternalFormId"\r\n' +
    '\r\n' +
    'survey_page_step\r\n' +
    '------WebKitFormBoundaryccYLQ3im9udgCOSx\r\n' +
    'Content-Disposition: form-data; name="CLIENT_ID"\r\n' +
    '\r\n' +
    'f68376f2931b4257838ba71f8df8b294\r\n' +
    '------WebKitFormBoundaryccYLQ3im9udgCOSx\r\n' +
    'Content-Disposition: form-data; name="survey.9b33f8ef-f905-49ad-bcd8-50248b61bf6a"\r\n' +
    '\r\n' +
    'Yes\r\n' +
...
}
```

### Expected Behavior

in 1.9.0 it would produce: 

{
  formQueryString: '__rvfInternalFormId=survey_page_step&CLIENT_ID=70843d75c0f34dbba27b2bc32d47b1d9&survey.9b33f8ef-f905-49ad-bcd8-50248b61bf6a=Yes&survey.6f4a11a9-f336-4d26-bf0c-d915aa7583db=No&survey.5512beac-3baa-438e-9218-d3b932df6bbb=Yes&survey.d8ea2fa5-8c8d-4459-9a5a-45036a78ac78=No&survey.ff990415-3ce7-4458-b152-658fad56937c=Yes&survey.ed45ad72-50cc-4598-9124-df906352fff0=No&survey.4891eee5-c333-4ce4-8e8f-310efc93ad8b=Yes'
}

### Actual Behavior

In 1.10-pre5 produces:

```js
{
  formQueryString: '------WebKitFormBoundaryccYLQ3im9udgCOSx\r\n' +
    'Content-Disposition: form-data; name="__rvfInternalFormId"\r\n' +
    '\r\n' +
    'survey_page_step\r\n' +
    '------WebKitFormBoundaryccYLQ3im9udgCOSx\r\n' +
    'Content-Disposition: form-data; name="CLIENT_ID"\r\n' +
    '\r\n' +
    'f68376f2931b4257838ba71f8df8b294\r\n' +
    '------WebKitFormBoundaryccYLQ3im9udgCOSx\r\n' +
    'Content-Disposition: form-data; name="survey.9b33f8ef-f905-49ad-bcd8-50248b61bf6a"\r\n' +
    '\r\n' +
    'Yes\r\n' +
...
}
```

**Comment by brophdawg11** [comment]
Thanks for the bug report!  This should be fixed in #4995 and we'll shoot to get another prerelease out tomorrow

**Comment by brophdawg11** [comment]
`1.10.0-pre.6` is out with the fix for this if you want to give it a spin!
